### PR TITLE
CIS Firewall Rules : added priority key 

### PR DESCRIPTION
**Pull request by arpit-srivastava-ibm**
Problem Statement - 
- User was not able to update `priority` field in Firewall Rules while creating or updating a resource

Issue - https://github.ibm.com/NetworkTribe/CIS_Support/issues/2515

### CODE REVIEW

**Comment by arpit-srivastava-ibm** [comment]
```
arpit-mac@Arpits-MacBook-Pro terraform-provider-ibm % make testacc TEST=./ibm/service/cis TESTARGS='-run=TestAccIBMCisFirewallrules_Basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./ibm/service/cis -v -run=TestAccIBMCisFirewallrules_Basic -timeout 700m
=== RUN   TestAccIBMCisFirewallrules_Basic
--- PASS: TestAccIBMCisFirewallrules_Basic (97.04s)
PASS
ok  	github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/cis	97.952s
```

**Comment by arpit-srivastava-ibm** [comment]
The test cases and documentation already covered creation and updation of resource with `priority` key.

File: ibm/service/cis/resource_ibm_cis_firewall_rules.go [hunk 1]
```diff
@@ -181,7 +185,8 @@ func ResourceIBMCISFirewallrulesUpdate(context context.Context, d *schema.Resour
 	if d.HasChange(cisFilterID) ||
 		d.HasChange(cisFirewallrulesAction) ||
 		d.HasChange(cisFirewallrulesPaused) ||
-		d.HasChange(cisFilterDescription) {
+		d.HasChange(cisFilterDescription) ||
+		d.HasChange(cisFirewallrulesPriority) {
```
Comment (hkantare) [inline ibm/service/cis/resource_ibm_cis_firewall_rules.go:190]: in update handle the support for change in priority

Comment (arpit-srivastava-ibm) [reply ibm/service/cis/resource_ibm_cis_firewall_rules.go:190]: Added

**Pull request merged**